nav/index.html

```diff
@@ -284,52 +284,54 @@ <h1 data-localize="GPS Denied">GPS Denied</h1>
 		var nav      = new Navigator(maths, lang_root);
 		var ui       = new UI(map, nav, settings, maths, lang_root);
 		var webapi   = new WebAPI(nav, ui, settings);
-		
+
 		// Navigation (map in review mode by default, without read GPS for not drain battery)
 		var id_gps = 0;
 		var confirm_route = true;
-		
+
 		function center_pos() {
 			ui.set_center_1st_pos(true);
 			ui.set_map_stretched(true);
 			if (id_gps === 0) // GPS will run forever
 				gps_start();
 		}
-		
-		
+
+
 		function gps_start() {
 
 			console.log('Starting GPS...');
-			
+
 			if (!navigator.geolocation) {
 				qml_set_center_onpos(0);
 				$('#gps_denied').show();
 				return;
 			}
-			
+
 			/** TESTING BEGIN **/
-			var route = [
-                [39.555550,3.087692], [39.555550,3.087692], [39.555550,3.087692], [39.555550,3.087692], [39.555550,3.087692], [39.556311,3.088636], [39.556311,3.088636], [39.556311,3.088636], [39.556311,3.088636], [39.557403,3.089365], [39.558230,3.090137], [39.558230,3.090137], [39.558230,3.090137], [39.558426,3.090643], [39.558757,3.090064], [39.559237,3.089978], [39.560146,3.089785], [39.561073,3.089592], [39.561668,3.089528], [39.561950,3.089634], [39.562099,3.090535], [39.562115,3.091135], [39.562264,3.091843], [39.562247,3.092530], [39.562281,3.093280], [39.562297,3.094031], [39.562496,3.095061], [39.562793,3.096004], [39.562959,3.096648], [39.563372,3.097463]
-			];
-			var z1 = 0;
-			id_gps = setInterval(function(){
-				accuracy = 300;
-				speed = 2;
-				z1++;
-				latitude = route[z1][0];
-				longitude = route[z1][1];
-				nav.set_gps_data(latitude, longitude, accuracy, speed);
-				gps_loop(nav.get_route_status());
-			},500);
-			return;
+			// var route = [
```

Comment:
Can we add here a condition? Or do you comment this part always before the release?

Reply:
> do you comment this part always before the release?

Yes, I have a local script for removing things like this one or change the API keys :)